zenodo/modules/pages/views.py

```diff
@@ -51,39 +52,54 @@
 )
 def support():
     """Render contact form."""
-    form_class = contact_form_factory()
-    form = form_class()
+    form = ContactForm()
+    uap = user_agent_information()
 
-    if form.include_os_browser.data:
-        form.os.data = request.user_agent.platform
-        form.browser.data = request.user_agent.browser
-        form.version.data = request.user_agent.version
+    # If user is logged in add email to it's form.
     if current_user.is_authenticated:
         form.email.data = current_user.email
 
-    """If form is validated send email to the admin."""
+    # Generation of message based upon maximum attachment size.
+    content = dict((a, c) for a, b, c in
+                   current_app.config['PAGES_ISSUE_CATEGORY'])
+
+    # Information added to form from config file.
+    form.issue_category.choices = [(row[0], row[1]) for row in
+                                   current_app.config['PAGES_ISSUE_CATEGORY']]
+    form.description.validators = [Length(
+        min=current_app.config['PAGES_DESCRIPTION_MIN_LENGTH'],
+        max=current_app.config['PAGES_DESCRIPTION_MAX_LENGTH'],
+    )]
+    form.attachments.description = 'Optional. Max attachments size: ' + \
+        do_filesizeformat(current_app.config['PAGES_ATTACHMENT_MAX_SIZE'])
+
+    # If form is validated send email to the admin.
     if form.validate_on_submit():
-        """Dictionary storing data to be sent."""
+        # Dictionary storing data to be sent.
         context = dict(
+            uap=uap,
             form=form,
             current_user=current_user,
```

Comment:
`current_user` can always be accessed through `flask_security.current_user`, so it's better not to pass it around, but use the import.